integration-tests/src/test/scala/tech/beshu/ror/integration/suites/base/BaseTemplatesSuite.scala

```diff
@@ -44,7 +44,7 @@ trait BaseTemplatesSuite
   private var originComponentTemplateNames: List[String] = List.empty
 
   protected def createIndexWithExampleDoc(documentManager: DocumentManager, index: String): Unit = {
-    adminDocumentManager.createFirstDoc(index, ujson.read("""{"hello":"world"}""")).force()
+    documentManager.createFirstDoc(index, ujson.read("""{"hello":"world"}""")).force()
```

Comment:
the previous one looked better

Reply:
The `documentManager` is passed to a method as a param and the method has more than 50 usages (for all cases `adminDocumentManager` is passed). I can remove `documentManager` from the param list, but IMO it won't improve readability